docs/verify/local-testing-mode.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Qiskit Runtime local testing mode
+description: Use local testing mode to simulate quantum hardware in Qiskit Runtime
+---
+
+<span id="test-locally"></span>
+# Qiskit Runtime local testing mode
+
+Quantum simulators should be used to help develop and test programs before fine-tuning them and sending them to real quantum hardware. Local simulators can do this with good performance and efficiency.  Clifford circuits can be simulated very efficiently, and results can be verified. This is useful to give confidence in an experiment.
+
+To use local testing mode with Qiskit Runtime, specify one of the fake backends in ``qiskit_ibm_runtime.fake_provider`` or specify a Qiskit Aer backend when instantiating a primitive or a session.  For details, see the [fake_provider API reference.](/api/qiskit-ibm-runtime/fake_provider)
```

Comment:
I think we should go into more details on Fake backends vs Aer, so people know when to use which. See my comments in the migration guide.